subprojects/docs/src/docs/release/notes.md

```diff
@@ -53,6 +53,14 @@ The following are the newly deprecated items in this Gradle release. If you have
 ### Example deprecation
 -->
 
+### Change of default Checkstyle configuration directory
+
+Configuring rules consumed by the Checkstyle plugin in a multi-project build used to be tedious.
```

Comment:
⭕️ I think we can drop the whole initial paragraph.. We're saying everything twice right now and might even confuse users with phrases like "users have to reconfigure".

Saying "it used to be tedious" is a bit over the top in my opinion, it was a one-line change to reconfigure it.

Reply:
done